playground/mocks/config/responseConfig.js

```diff
@@ -755,6 +755,14 @@ const googleAuthenticatorVerify = {
   ],
 };
 
+const pivAuth = {
+  '/idp/idx/introspect': [
+    'identify-with-third-party-idps',
+    //'error-identify-with-piv',
```

Comment:
why do we comment it out?

Reply:
This is mock data. So for introspect i first test with the first response. When i need to test other scenarios, i uncomment those responses